_layouts/recipe.html

```diff
@@ -31,10 +31,12 @@
 </head>
 <body class="pg-tutorial">
   {% include nav.html %}
-
   <section class="hero">
     <div class="hero-body">
       <div class="container">
+        <ul class="breadcrumb">
+          <li><a href="/tutorials/use-cases">&lt; Back to tutorials</a></li>
```

Comment:
Should this be use-cases.html?  (Note addition of `.html`)

Reply:
Updated.!